app/presenters/katello/sync_status_presenter.rb

```diff
@@ -0,0 +1,92 @@
+module Katello
+  class SyncStatusPresenter
+    include ::ActionView::Helpers::DateHelper
+    include ::Katello::TranslationHelper
+    STATUS_VALUES = { 
+                       :stopped      => _("Syncing Complete."),
+                       :error        => _("Sync Incomplete"),
+                       :never_synced => _("Never Synced"),
+                       :running      => _("Running"),
+                       :canceled     => _("Canceled"),
+                       :paused       => _("Paused")
+                    }.with_indifferent_access
+
+    class Status
```

Comment:
@ehelms during katello:reindex,
got an undefined method 'state' in `sync_state` ( https://github.com/Katello/katello/blob/cfddcb409c1a2a2fd04fd28ad880d870e1b74492/app/models/katello/glue/provider.rb#L77 ) when re-indexing providers. It seemed to be calling the sync_satus, in this change set (https://github.com/Katello/katello/pull/5234/files#diff-c72291fac84a47389f1d4fe5125e76a3L177) instead of the one defined in `Glue::Provider`.